addon/routes/edit-map.js

```diff
@@ -118,5 +137,36 @@ export default EditFormRoute.extend({
     }
 
     return result;
+  },
+
+  /**
+    Recursively creates layers hierarchy.
```

Comment:
are comment correspond to method work? it seems you ctrl+c\ctrl-v from sortLayersByIndex.